packages/reactive-dict/reactive-dict.js

```diff
@@ -94,19 +103,19 @@ _.extend(ReactiveDict.prototype, {
     // XXX we could allow arrays as long as we recursively check that there
     // are no objects
     if (typeof value !== 'string' &&
-        typeof value !== 'number' &&
-        typeof value !== 'boolean' &&
-        typeof value !== 'undefined' &&
-        !(value instanceof Date) &&
-        !(ObjectID && value instanceof ObjectID) &&
-        value !== null)
+      typeof value !== 'number' &&
```

Comment:
Sorry.  Before I do another pull-request, I'll sit down and give my prettify script a close audit and make sure that it's in alignment with Meteor styleguides.  I ran it to add spaces around language keywords, and it wound up being one step forward and one step backwards.